test-cloud/pom.xml

```diff
@@ -236,9 +257,6 @@
     </profile>
     <profile>
       <id>openshift</id>
-      <properties>
```

Comment:
What is the reason to move these properties configurations to submodules?
It just causes duplication from my point of view.

Reply:
SkipTests property from this module was ignored and all tests were skipped.